examples/textual_inversion/textual_inversion.py

```diff
@@ -541,9 +541,10 @@ def main():
     unet.to(accelerator.device, dtype=weight_dtype)
     vae.to(accelerator.device, dtype=weight_dtype)
 
-    # Keep vae and unet in eval model as we don't train these
-    vae.eval()
```

Comment:
`vae` is already in `eval` mode by default, when loaded using `from_pretrained`.